airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV1.py

```diff
@@ -137,7 +137,7 @@ class Data(BaseModel):
     license: str
     supportUrl: AnyUrl
     githubIssueLabel: str
-    sourceType: str
+    connectionType: str
```

Comment:
Ya, looks like git made an incorrect looking diff 😅 

However if you have better names for `connectorType` and `connectionType` im all ears... not happy with how close they are. Feels like the Dagger vs Dagster name problem

Reply:
`connectionType` is in use

Before it was called `sourceType` and used only for sources to inidicate whether it was a `api`, `file` or `database` source.

But this updates `sourceType` to now be called `connectionType` allowing destinations to do the same self reporting

Reply:
Ah, I'm recommend we call it `connectorType` instead! Since that's what we use elsewhere and I think would be the correct term. Or do you think of `connectorType` as referring to source/destination vs `connectionType` being api/file/database?

Reply:
Thats the tricky part! `connectorType` is being used just like you described

>  connectorType as referring to source/destination vs connectionType being api/file/database

Reply:
Aha. Word! Okay, so yeah, a dagger/dagster situation. 

How about something like `typeTags`? I'm imagining your could have multiple if we ended up defining more (e.g. `[api, python]`) - this feels less like a hard classification and more like tagging for extra context. 

Or even `connectorSubtype` or something of the sort? I feel like we might end up using this to convey more than just the `://` connection type... but I could be wrong!

Reply:
Im into that! but! we do program on this field in cloud if I recall. Basically the value corresponds to the same column that can have the value `custom` which then the platform has a few lines of code sprinkeled around the codebase that go `if sourceType === 'custom'`

Its not ideal, but Im currently thinking we dont open that refactor can of worms, by changing this to an array.

Yet 😅 

But connectorSubtype works! I like that!

Reply:
Cool, yeah a refactor is unnecesary at the moment, but definitely something to think about! Good to know we're actually using this field at the moment